src/dlgConnectionProfiles.cpp

```diff
@@ -651,7 +655,7 @@ QString dlgConnectionProfiles::getDescription(const QString& hostUrl, const quin
                 "not enforced, is rewarded with non-PvP oriented perks and unique treasure.\n\n"
                 "A powerful character customization system allows you to choose your deity –or fully disregard the gods– and join one of the player-run realms that govern the land "
                 "to explore a breathing world, delve into the secrets of the oceans, shape your legacy, craft forgotten marvels for you –or your allies– and fight for faith, glory or coin.");
-                /**
+        /**
```

Comment:
Humm, has the `.clang-format` changed recently - I've noticed that some long C style comments are getting _adjusted_ like this...

Reply:
Nope... but is this a problem or... ?

Reply:
More like a `niggle` - it means that using the Qt clang-format _beautifier_ on the whole file is going to mess such comments up - not in a destructive way - they are still going to work as comments but it does impede keeping the style as we would like...

Anyhow - it isn't a blocker for this PR but something to nail down in the specifications in the `.clang-format` file and then run the formatter over the affected parts of the affected files...

Reply:
Okay - mind approving this PR then?

Reply:
>  SlySven approved these changes on behalf of Mudlet/core-cpp an hour ago 

Um, I already had...